src/contracts/middleware/VoteWeigherBase.sol

```diff
@@ -60,12 +69,7 @@ contract VoteWeigherBase is VoteWeigherBaseStorage {
 
                 // add the weight from the shares for this strategy to the total weight
                 if (sharesAmount > 0) {
-                    weight += uint96(
-                        (
-                            (strategyAndMultiplier.strategy).sharesToUnderlying(sharesAmount)
-                                * strategyAndMultiplier.multiplier
-                        ) / WEIGHTING_DIVISOR
-                    );
+                    weight += uint96(sharesAmount * strategyAndMultiplier.multiplier / WEIGHTING_DIVISOR);
```

Comment:
discussed this offline and agreed it was an acceptable change.
I also think this _could_ be abstracted into a 'hook' or similar

Reply:
going to note here that I think adding an `@dev` tag to this function that explains the side-effects/risk of this function ignoring fluctations in the conversion rate of shares to underlying

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/53/commits/07031237ae72de11a7507e3ff3bb2f1e08ea9eb4